monero-outreach-docs/translations/archive/wallet_primer_fr.md

```diff
@@ -0,0 +1,90 @@
+
+### _Contexte Technique_
+Il est important d’avoir une représentation mentale sur la manière dont un portefeuille papier de Monero fonctionne afin de se sentir à l’aise et d’utiliser le dit portefeuille correctement. La clé de dépense privé mentionnée précédemment, ressemble à quelque chose comme cela sous forme hexadécimale (utilisant 0-9 et a-f):
+
+_Clé de dépense privée :_
+**09f8225e9f6dd95457a0bca31b66a599 6e6da8be8195025817c36bf490ae0903**
+
+Représentant de manière brute 256 bits arbitraire d’information, elle a beaucoup de caractères et est si compliqué que personne ne peut la deviner. Vous pouvez juste l’écrire et la cacher, et alors vous possédez les Monero qu’elle contrôle.
+Par ce que la clé de dépense privée à l’état brute est complexe et est sujette à erreurs, des représentations ont été créés sous forme de groupes de mots habituels, plus familiers, appelée phrase mnémonique. Les deux plus importantes représentations pour Monero sont les suivantes :
+
+**Représentation 1  (25 mots) :** La représentation intégré au portefeuille Monero officiel utilise 25 mots ordonnées choisis à partir d’un ensemble de 1626 mots. Ils tiennent compte de contraintes internes de tel sorte qu’un choix arbitraire de 25 mots est généralement invalide. Ceci permet au logiciel de détecter des erreurs. La liste des mots anglais peut être vu ici : : [github.com/monero-project/monero/blob/master/src/mnemonics/english.h](github.com/monero-project/monero/blob/master/src/mnemonics/english.h). La représentation mnémonique pour la clé de dépense privé montré ci-dessus est la suivante : _{moon pram oust mime boil boat nutshell moisture ionic enjoy below ungainly loaded rage etched atom reduce almost glass vexed jaded coils pioneer pyramid nutshell}_.
+
+**Représentation 2 (24 mots) :** La représentation standard de la suggestion d’amélioration de Bitcoin 39 (BIP39) est une représentation générique utilisée, par exemple, par le portefeuille matériel Ledger. Il est composé de 24 mots ordonnés, choisis parmi un ensemble de 2048 mots. De manière similaire à la représentation 1, cette représentation répond à des contraintes internes de telle sorte que la plupart des combinaisons de mots sont invalides. La liste de mot anglais peut être vu ici : [github.com/bitcoin/bips/blob/master/bip-0039/english.txt](github.com/bitcoin/bips/blob/master/bip-0039/english.txt). Un exemple de phrase mnémonique de ce type est le suivant : _{open birth pepper electric nuclear shine stable ritual napkin quarter aunt demand thing cable tuna lucky vehicle taste tourist grape buffalo fitness indoor draft}_.
```

Comment:
For 1. The original is "The standard Bitcoin Improvement Proposal 39 (BIP39)"
I agree that the translation is a bit strange but it is a bit tough to translate accurately. May be adding "numéro" would sound better?  This would give: **la suggestion d’amélioration de Bitcoin numéro 39**
Are you OK with that?

Reply:
And the original file is there: https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/en/archive/wallet_primer_en.md